Changes based on opsdir review (Joel Jaeggli) (#27)
core-wg/core-problem-details: draft-ietf-core-problem-details.md

`custom-problem-detail-entries` slot according to the procedure described in
{{new-cpdk}}.

Consumers of a Concise Problem Details data item MUST ignore any
Standard or Custom Problem Detail entries, or keys inside the Custom
Problem Detail entries, that they do not recognize ("ignore-unknown
rule"); this allows problem details to evolve.
When storing the data item for future use or forwarding it to other
consumers, it is strongly RECOMMENDED to retain the unrecognized
entries; an exception might be when storage/forwarding occurs in a

> different format/protocol that cannot accommodate them.

Reviewer: another exception is that of a node that needs to avoid to accidentally forward potentially privacy-sensitive info.

Author: Addressed in 76fe25d

entry, the documentation of which defines these internal entries,
typically chosen to address a given application domain.

> When evolving Custom Problem Detail entry definitions, the
> "ignore-unknown rule" discussed in the introduction to
> {{sec-new-attributes}} can help with including additional information.

Reviewer: It sounds like an pretty intricate way of just saying:
```suggestion
The "ignore-unknown rule" discussed in the introduction to
{{sec-new-attributes}} applies fully.
```
no? 😄

Author: Well, this sentence is a lead-in to specifically discussing ignore-unknown *inside* a CPDe.
So I don't think the simplified form helps as much.

Reviewer: Removing _"fully"_ from the suggested text could do the trick?

Stepping back a bit: I have troubles making sense of _"[...] can help with including additional information."_.

I think what we are trying to tell the reader is: "apply ignore-unknown when dealing with extensions found in CPDe".  If so, the way the text flows is not conducive to quick and unambiguous grasp of the concept.

Author: I think we are trying to explain you don't have to overdo the evolution of a CPDe because you always can make a new one.

Reviewer: Then we probably should say exactly so :-)

Author: I think we do.  Note that this is text that is directed at the person who wants to evolve a CPDe definition; if it were about handling evolved definitions, it would indeed be redundant.

I tried to make this context more accessible in adb7076
